arch/arm/src/sama5/sam_tsd.h

```diff
@@ -36,10 +36,15 @@
 
 /* Configuration ************************************************************/
 
-#ifdef CONFIG_SAMA_TSD_RXP
+#ifndef CONFIG_SAMA_TSD_RXP
 #  define CONFIG_SAMA_TSD_RXP 6
 #endif
 
+/* Only allow Pendet triggering in limited circumstances */
```

Comment:
Add space after comment

Reply:
Surprised that checkpatch.sh missed that, but OK, of course